source/server-selection/server-selection-tests.rst

```diff
@@ -296,20 +298,28 @@ Multi-threaded and async drivers MUST also implement the following prose test:
         },
     }
 
-3. Create a client with both mongoses' addresses in its seed list,
-   appName="loadBalancingTest", and command monitoring enabled.
+3. Insert an empty document into the collection being used for the test.
 
-4. Start 10 concurrent threads / tasks that each run 10 `findOne` operations
-   using that client.
+4. Create a client with both mongoses' addresses in its seed list,
+   appName="loadBalancingTest", and localThresholdMS=30000.
```

Comment:
the high localThresholdMS should help avoid cases where a single mongos happens to get all the selections due to a random spike in network latency.

Reply:
Do you think it would be helpful to include these notes in the actual text for future implementers/to help troubleshoot failures?

Reply:
Good idea, added